### PR TITLE
NXP-26685: meaningful path in nuxeo-document-publications

### PR DESCRIPTION
**Pull request by semisse**

### CODE REVIEW

**Comment by semisse** [comment]
To avoid the problem that is referenced on ELEMENTS-1002, we are using here a `&lrm;` as workaround, until we find a better solution to handle these cases.

File: elements/nuxeo-publication/nuxeo-document-publications.js [hunk 1]
```diff
@@ -46,12 +46,21 @@ Polymer({
...
+        direction: rtl;
+        text-align: left;
+      }
+
+      .right-ellipsis {
+        text-align: right;
+      }
+
       .ellipsis {
+        width: calc(100% - 38px);
```
Comment (nmpcunha) [inline elements/nuxeo-publication/nuxeo-document-publications.js]: any specific reason to change the order?

File: elements/nuxeo-publication/nuxeo-document-publications.js [hunk 2]
```diff
@@ -117,7 +121,9 @@ Polymer({
         <nuxeo-data-table-column name="[[i18n('documentContentView.datatable.header.path')]]" flex="200">
           <template>
             <nuxeo-document-thumbnail document="[[item]]"></nuxeo-document-thumbnail>
-            <a class$="path ellipsis [[_ellipsisDirection()]]" href$="[[urlFor('browse', item.path)]]">[[item.path]]</a>
+            <a class$="path ellipsis [[_ellipsisDirection()]]" href$="[[urlFor('browse', item.path)]]"
+              >[[item.path]]&lrm;</a
+            >
```
Comment (nmpcunha) [inline elements/nuxeo-publication/nuxeo-document-publications.js]: Why was it different from the first commit? Was it lint?

File: elements/nuxeo-publication/nuxeo-document-publications.js (hunk 1 above)
Comment (semisse) [reply elements/nuxeo-publication/nuxeo-document-publications.js]: This doesn't have a particular impact in the visuals, it is just regarding the discussion of having some consistency on listing the CSS properties (and using the Box Model to do it).

File: elements/nuxeo-publication/nuxeo-document-publications.js (hunk 2 above)
Comment (semisse) [reply elements/nuxeo-publication/nuxeo-document-publications.js]: Yes, probably the introduction of `&lrm;` exceeded the number of characters allowed in one line.

Comment (nmpcunha) [reply elements/nuxeo-publication/nuxeo-document-publications.js]: Of course... 🤦‍♂

File: elements/nuxeo-publication/nuxeo-document-publications.js [hunk 3]
```diff
@@ -33,6 +33,10 @@ import { html } from '@polymer/polymer/lib/utils/html-tag.js';
 Polymer({
   _template: html`
     <style include="nuxeo-styles">
+      [hidden] {
+        display: none !important;
```
Comment (nmpcunha) [inline elements/nuxeo-publication/nuxeo-document-publications.js]: Do we need to force with `!important`?

File: elements/nuxeo-publication/nuxeo-document-publications.js (hunk 1 above)
Comment (nmpcunha) [reply elements/nuxeo-publication/nuxeo-document-publications.js]: Only impact on PR diff visual 😜 
I foresee the reason and I'm not against it, but please recall that the discussion didn't took place yet and there is still no agreement on that guideline.

File: elements/nuxeo-publication/nuxeo-document-publications.js [hunk 4]
```diff
@@ -154,7 +160,7 @@ Polymer({
             </template>
           </template>
         </nuxeo-data-table-column>
-        <nuxeo-data-table-column flex="10">
+        <nuxeo-data-table-column flex="10" hidden$="[[!_canRepublish()]]">
```
Comment (nmpcunha) [inline elements/nuxeo-publication/nuxeo-document-publications.js]: This method expects an input, correct?
Also, what would be the visual impact when having multiple documents, some that can be republished, some that cannot?

**Comment by semisse** [comment]
We've ditched the idea of checking if the document is republishible or unpublishible because it might be tricky (in case the document has hundreds of versions, for example).

**Comment by semisse** [comment]
Pipeline ok: https://qa.nuxeo.org/jenkins/job/web-ui-pipeline-sfialho/

**Pull request merged**